packages/relay/src/lib/eth.ts

```diff
@@ -1756,7 +1756,7 @@ export class EthImpl implements Eth {
         blockNumber: cachedLog.blockNumber,
         contractAddress: cachedLog.address,
         cumulativeGasUsed: EthImpl.zeroHex,
-        effectiveGasPrice: EthImpl.zeroHex,
+        effectiveGasPrice: await this.gasPrice(requestIdPrefix),
```

Comment:
My understanding is that this is the `current` gasPrice, when we should be returning the `gasPrice` at the time of the transaction.